### PR TITLE
#1060 added test for slack radar

### PR DESCRIPTION
**Pull request by krzyk**
#1060 
* added test for slack radar

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1222](https://github.com/zerocracy/farm/issues/1222) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1222](https://github.com/zerocracy/farm/issues/1222) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1741388550); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/zerocracy/radars/slack/SlackRadarTest.java [hunk 1]
```diff
@@ -58,6 +57,24 @@ public void closesSession() throws IOException {
         Mockito.verify(session).disconnect();
     }
 
+    @Test
+    @SuppressWarnings("unchecked")
```
Comment (carlosmiranda) [inline src/test/java/com/zerocracy/radars/slack/SlackRadarTest.java]: @krzyk where is the warning being thrown? It's not clear from the code itself.

Comment (krzyk) [reply src/test/java/com/zerocracy/radars/slack/SlackRadarTest.java]: @carlosmiranda it is in ` Mockito.mock(Reaction.class),` - Reaction class is a generic one and Mockito has not sane method to mock those without the warning

**Comment by krzyk** [comment]
@carlosmiranda replied, please see above

File: src/test/java/com/zerocracy/radars/slack/SlackRadarTest.java (hunk 1 above)
Comment (carlosmiranda) [reply src/test/java/com/zerocracy/radars/slack/SlackRadarTest.java]: @krzyk all right, makes sense

**Comment by krzyk** [comment]
@carlosmiranda do you have more comments here?

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1222](https://github.com/zerocracy/farm/issues/1222) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@carlosmiranda According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

No issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by carlosmiranda** [comment]
@ypshenychka confirmed

**Comment by ypshenychka** [comment]
@carlosmiranda Thank you

**Comment by ypshenychka** [comment]
@0crat quality bad

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**